vizro-core/src/vizro/tables/dash_aggrid.py

```diff
@@ -4,10 +4,12 @@
 
 
 @capture("action")
-def dash_ag_grid(data_frame=None):
+def dash_ag_grid(data_frame=None, **kwargs):
     """Custom AgGrid."""
     return dag.AgGrid(
         id="get-started-example-basic",
         rowData=data_frame.to_dict("records"),
-        columnDefs=[{"field": col} for col in data_frame.columns],
+        columnDefs=[{"field": col, "filter": True} for col in data_frame.columns],
+        dashGridOptions={"rowHeight": 40},
```

Comment:
The row height can be set in the .css file with  `--ag-row-height`

Reply:
I just checked locally, and you can use --ag-row-height as AnnMarie suggested, but you have to delete #page-container from `#page-container .ag-theme-alpine.vizro` in order for it to work.  

If deleting #page-container breaks something in the styling, you can use `height: 40px !important;` in `#page-container .ag-theme-alpine .ag-header-row`.  (unfortunately, it will not work without `!important` tag). Everything seemed okay once I delete it, but do double check if you opt for this option.

Reply:
I think we should avoid using the `!important` tag as it makes it impossible for the user to change it. You can still achieve a row-height of 40px by using some of the CSS selectors that are not `.ag-theme-alpine.vizro`, but more specific than that 👍

Reply:
I just checked, and you are correct @nadijagraca . The row height can be adjusted when removing `#page-container`.

Reply:
So this now works, but the header height is strange.

I am setting it explicitly, it seems not overwritten, but also it seems to have no effect

Reply:
The header height is a different selector I think.

Reply:
I thought I found it (see [here](https://github.com/mckinsey/vizro/pull/268/files#r1469886327))

Reply:
The header height can be set with 

```
  --ag-row-height: 48px; /* This determines the row height, but not header */
  --ag-header-height: 40px; /* This determines the header height */
```

More info here:  https://www.ag-grid.com/react-data-grid/global-style-customisation-headers/

Reply:
I attempted this, but it seems to have no effect. For instance when I set `--ag-header-height: 200px;` this is not visible.

Reply:
I'm not sure what to say.  I added the following as line 11 in the  vizro-core/src/vizro/static/css/aggrid.css and it worked just fine.  It was a little hard to see because the height of the grid in the demo app is so small but I could definitely change the header height 
```
--ag-header-height: 200px
```

Reply:
I must have done something silly or not updated properly! I literally just did the exact same again it it works... Apologies for the confusion! 🙈